presto-docs/src/main/sphinx/functions/geospatial.rst

```diff
@@ -316,6 +316,10 @@ Accessors
     Returns the last point of a LineString geometry as a Point.
     This is a shortcut for ST_PointN(geometry, ST_NumPoints(geometry)).
 
+.. function:: ST_Points(Geometry) -> array(Point)
```

Comment:
In the docs functions are listed in alphabetical order. ST_Points goes right after ST_PointN.